packages/store-sync/src/recs/syncToRecs.ts

```diff
@@ -59,9 +64,48 @@ export async function syncToRecs<TConfig extends StoreConfig = StoreConfig>({
 
   world.registerDisposer(stopSync);
 
+  const functionSelectorToResourceSelector = new Map<Hex, Hex>();
```

Comment:
it might make sense to keep this function not-cached and move the caching of it into `createContract`